### PR TITLE
Option to not separate numbers within words

### PR DESCRIPTION
**Pull request by arjanfrans**
Issue: https://github.com/lovell/limax/issues/14

### CODE REVIEW

File: lib/limax.js [hunk 1]
```diff
@@ -6,14 +6,24 @@ var hepburn = require('hepburn');
 
 module.exports = function(text, opt) {
   var options = opt || {};
+  var separateNumbers = true;
+
   if (typeof options === 'string') {
     options = {separator: options};
   }
 
+  if (typeof options.separateNumbers !== 'undefined') {
```
Comment (lovell) [inline lib/limax.js]: Would `typeof === 'boolean'` be better here?

**Comment by lovell** [comment]
Thank you! I've added one minor nit-pick inline, otherwise this looks great.

File: lib/limax.js (hunk 1 above)
Comment (arjanfrans) [inline lib/limax.js]: If you want to enforce a boolean, yes. With !== undefined you can put in any truthy or falsy value.

Feel free to change it.

**Comment by lovell** [comment]
Thanks again, I'll get a release with your addition published soon.

**Pull request merged**